assembl/static2/js/app/utils/translations.js

```diff
@@ -560,7 +561,8 @@ const Translations = {
       contact: "Do you wish to know more about this platform?",
       contactUs: "Contact us",
       from_start_to_end: "from %{start} to %{end}",
-      chatbot: "Chat with %{chatbotName}"
+      chatbot: "Chat with %{chatbotName}",
+      assemblNotConfigured: "Assembl is not configured properly"
```

Comment:
I wouldn't say "not configured properly" but rather "has not been fully configured yet"

Reply:
done